src/phoenix/session/client.py

```diff
@@ -425,14 +428,97 @@ def download_dataset_examples(
             index_col="example_id",
         )
 
+    def create_examples(
+        self,
+        *,
+        name: str,
```

Comment:
I like the simplicity but it took me a second that name was the name of the dataset and the description was the dataset description as there would be a version too.

Reply:
Good catch! I'll clean up. Thanks

Reply:
I don't think version is needed since that's created on the server side.